ui/public/i18n/language.json

```diff
@@ -22,9 +22,20 @@
   },
   "settings": {
     "title": "Settings",
-    "configure_instance": "Configure {instance}",
+    "piler_fqdn": "piler FQDN",
+    "imap_fqdn": "IMAP server FQDN (user authentication)",
```

Comment:
Can we shorten this label and make it a bit more clear? I'd remove `(user authentication)` and explain it inside a tooltip